doc/source/examples/extended_examples/sfem/sfem.py

```diff
@@ -0,0 +1,758 @@
+    from pathlib import Path
+
+    from ansys.mapdl.core import launch_mapdl
+
+    path = Path.cwd()
+    mapdl = launch_mapdl(run_location=path)
+
+    domain = [0, length]
+    correl_length_param = 3
+    min_eigen_value = 0.001
+    poisson_ratio = 0.3
+
+    mapdl.prep7()  # Enter preprocessor
+
+    mapdl.r(r1=thickness)
+    mapdl.et(1, "PLANE182", kop3=3, kop6=0)
+    mapdl.rectng(0, length, 0, height)
+    mapdl.mshkey(1)
+    mapdl.mshape(0, "2D")
+    mapdl.esize(mesh_size)
+    mapdl.amesh("ALL")
+
+    # Fixed edge
+    mapdl.nsel("S", "LOC", "X", 0)
+    mapdl.cm("FIXED_END", "NODE")
+
+    # Load application node
+    mapdl.nsel("S", "LOC", "X", length)
+    mapdl.nsel("R", "LOC", "Y", height)
+    mapdl.cm("LOAD_APPLICATION_POINT", "NODE")
+
+    mapdl.finish()  # Exit preprocessor
+
+    mapdl.slashsolu()  # Enter solution processor
+
+    element_ids = mapdl.esel(
+        "S", "CENT", "Y", 0, mesh_size
+    )  # Select bottom row elements and store the ids
+
+    # Generate quantities required to define the Young's modulus stochastic process
+    cosine_frequency_list, cosine_eigen_values, cosine_constants = (
+        evaluate_KL_cosine_terms(domain, correl_length_param, min_eigen_value)
+    )
+    sine_frequency_list, sine_eigen_values, sine_constants = evaluate_KL_sine_terms(
+        domain, correl_length_param, min_eigen_value
+    )
+
+    simulation_results = np.zeros(no_of_simulations)
+
+    for simulation in range(no_of_simulations):
+        # Generate random variables and load needed for one realization of the process
+        cosine_random_variables_set = np.random.normal(
+            0, 1, size=len(cosine_frequency_list)
+        )
+        sine_random_variables_set = np.random.normal(
+            0, 1, size=len(sine_frequency_list)
+        )
+        load = -np.random.normal(10, 2**0.5)  # Generate a random load
+
+        material_property = 0  # Initialize material property ID
+        for element_id in element_ids:
+            material_property += 1
+            mapdl.get("ELEMENT_ID", "ELEM", element_id, "CENT", "X")
+            element_centroid_x_coord = mapdl.parameters["ELEMENT_ID"]
+            mapdl.esel(
+                "S", "CENT", "X", element_centroid_x_coord
+            )  # Select all elements having this coordinate as centroid
+
+            # Evaluate young's modulus at this material point
+            young_modulus_value = young_modulus_realization(
+                cosine_frequency_list,
+                cosine_eigen_values,
+                cosine_constants,
+                cosine_random_variables_set,
+                sine_frequency_list,
+                sine_eigen_values,
+                sine_constants,
+                sine_random_variables_set,
+                domain,
+                element_centroid_x_coord,
+            )
+
+            mapdl.mp(
+                "EX", f"{material_property}", young_modulus_value
+            )  # Define property ID, assign young's modulus
```

Comment:
```suggestion
            )  # Define property ID and assign Young's modulus
```
Even comments should use proper capitalization.